src/transformers/models/superpoint/image_processing_superpoint.py

```diff
@@ -270,3 +273,28 @@ def preprocess(
         data = {"pixel_values": images}
 
         return BatchFeature(data=data, tensor_type=return_tensors)
+
+    def post_process_keypoint_detection(self, outputs, target_sizes, unwrap_batch_dim=True):
```

Comment:
We should add this as a documented method for the model's doc page e.g. [like here ](https://github.com/huggingface/transformers/blob/c409cd81777fb27aadc043ed3d8339dbc020fb3b/docs/source/en/model_doc/deformable_detr.md?plain=1#L61)